ScienceJournal.xcodeproj/project.pbxproj

```diff
@@ -3040,7 +3040,7 @@
 					68EBA6D91E20421E0089FF7F = {
 						CreatedOnToolsVersion = 8.2.1;
 						DevelopmentTeam = EQHXZ8M8AV;
-						LastSwiftMigration = 1000;
+						LastSwiftMigration = 1110;
```

Comment:
Revert pbxproj changes to avoid the Swift 5 upgrade here if possible?

Reply:
Confirmed we're already on Swift 5, so this is just to silence the upgrade warnings 🎉